services/user.js

```diff
@@ -85,14 +87,23 @@ const userService = {
     return newUser
   },
 
-  updateUser: async (id, param) => {
+  privateUpdateUser: async (id, param) => {
     const user = await User.findByIdAndUpdate(id, param, { new: true }).exec()
 
     if (!user) {
       throw createError(404, USER_NOT_FOUND)
     }
   },
 
+  updateUser: async (id, param) => {
+    const filteredParam = filterAllowedFields(param, allowedUserFieldsForUpdate)
```

Comment:
from what i see only status will be available to update,
so prolly no sense to add here filteredParam or whatever, can directly pass status and thats it

Reply:
but it doesnt mean that you should delete filteredParam, leave it for different endpoint

Reply:
changed this service only for updating status by admin, and also added to models updated field status, you can check